### DISCUSSION TITLE
Issues facing while requesting for Swags

### DISCUSSION THREAD

**Discussion by mridungupta1984** [community/community#46694] [GitHub Education] [answered]
Hey, Community Members and @github-staff.
---
**Today, I am starting this discussion to report for the issues that multiple users are facing (while submitting request for Swags).**

1. **_Option is for requesting is no longer visible_**
> Screenshot of the "Global Campus" for reference
> <img src="https://user-images.githubusercontent.com/119317635/217423932-22874a8d-9a2b-40c1-95cf-934c58cbfecf.png" width="500" alt="issue-1_screenshot">
> [by @mridungupta1984]<br>
2. **_Unauthorized users at the [link](https://githubedu.printengine.com/product/github-teacher-toolbox-swag-offer/) {general URL for requesting swags}_**
> Screenshot of the error occurring (`Courtesy to some users participated for other Github Discussions`)<br>
> <img src="https://user-images.githubusercontent.com/119317635/217425535-118ac823-310f-4a00-8a38-09efc3f63036.png" width="500" alt="issue-2_screenshot1">
> [by @endersonmenezes]<br>
> <img src="https://user-images.githubusercontent.com/45125059/152116951-28e70331-84a4-4788-b10f-bb0cdd1ca134.png" height="500" alt="issue-2_screenshot2">
> [by @abhishekkumkar]

**Comment by mridungupta1984** [comment]
I request @github-staff to look into this at the earliest. [@samus-aran or @divais]

**Reply by mzunnurainhussain** [reply]
Hi, Can i get a link to apply for teachers swag please. I can see it it was to be launced in May23.

**Comment by t-abraham** [comment]
I confirm that I am also having the same issue. Tried with all possible platforms and browsers.

**Comment by gavinahales** [comment]
I can get to the printengine order page but after entering my details and pressing Place Order, a black error box appears at the top with no text.

**Comment by qu-jieniu** [comment]
Thanks for creating this thread! 

I'm having the same issue, but with a little different wordings (i.e., Your Github account isn't affiliated with the education platform.)

Please see attached pic showing i was approved as a teacher, and the screenshot from printengine.

![Screen Shot 2023-02-08 at 12 03 22 PM](https://user-images.githubusercontent.com/96959297/217601123-e5e752f7-c483-41b5-bde7-f5def1cc87d7.png)
![Screen Shot 2023-02-08 at 12 03 45 PM](https://user-images.githubusercontent.com/96959297/217601153-90a67135-faea-4e5d-91a8-cc7506d3de5f.png)

**Comment by ghost** [comment]
@github-staff same problem.

**Comment by mridungupta1984** [comment]
![image](https://user-images.githubusercontent.com/119317635/218113577-6a0a87b2-530a-44f9-9a2f-859293bcfa95.png)
---
I request @divais to confirm the same here.

**Comment by samus-aran** [comment, marked as answer]
Hi all Global Campus Educators! 👋

We apologize for the recent hiccup with our Teacher Swag Request platform. Unfortunately, a bug was discovered with our third-party vendor and we had to pause the request link temporarily to investigate the issue. We know this has caused frustration and we apologize for not communicating this sooner.

We're working to resolve the issue and reinstate the request link as soon as possible. This may take a few weeks, but our goal is to process all existing requests.

Thank you for your understanding and continued support of GitHub Education.

**Reply by mridungupta1984** [reply]
Thank you, @samus-aran for the confirmation.

**Reply by gauravGo7** [reply]
Hi all Global Campus Educators! 👋

We apologize for the recent hiccup with our Teacher Swag Request platform. Unfortunately, a bug was discovered with our third-party vendor and we had to pause the request link temporarily to investigate the issue. We know this has caused frustration and we apologize for not communicating this sooner.

We're working to resolve the issue and reinstate the request link as soon as possible. This may take a few weeks, but our goal is to process all existing requests.

Thank you for your understanding and continued support of GitHub Education.

**Reply by mridungupta1984** [reply]
> Fixed?

Not yet, you have to wait (@fornewtrailway)

**Reply by ashwin3082002** [reply]
Hey @samus-aran, I am trying to order swags from the admin account (Github Campus Program), But I am not able to do it as it is showing an unauthorized user even after using the super admin account. Is the issue resolved?

@github-staff

**Reply by yondoi** [reply]
@mridungupta1984 are you there

**Reply by Sir-karan-singh** [reply]
@samus-aran any updates? May is
 over already

**Reply by turlakriju** [reply]
@samus-aran update

**Reply by mridungupta1984** [reply]
> @mridungupta1984 are you there

Yes, How may I help you?

**Reply by mridungupta1984** [reply]
@github-staff @github-education @samus-aran any update yet?

**Reply by umerslone** [reply]
Hi- any update?

**Reply by mridungupta1984** [reply]
Nope.

**Reply by mridungupta1984** [reply]
Ah! Any other @github-education might answer other than @samus-aran.

**Reply by mohsinmehdi** [reply]
I still cannot request swag offer. Any update on this?

**Comment by MoiGonzagaUTT** [comment]
My current issue is that it asks me how many students I have but doesn't allow me to enter and continue the process. I will wait on any feedback in the coming days. But thank you so much for your continuing support on this.

![image](https://user-images.githubusercontent.com/82000191/220183412-26fe3fee-8745-44ab-a350-666ee9f3e3a3.png)

**Reply by petrvecera** [reply]
I am getting the same error. I Was able to get the swag bag for all my previous courses (2021, 2022) but this year I am facing the issues :'(

**Comment by mridungupta1984** [comment]
The coupon that **Imprint Engine** is assigning to the request made under _GitHub Swag Program_ are found to be having some glitches that they are not usable on [The Github Shop](www.thegithubshop.com). I request someone from @github-staff or the Imprint Engine team to look into this.

**Reply by mridungupta1984** [reply]
@samus-aran issue still not reviewed yet, please look for this matter before upcoming scheduled revamping.

**Comment by gajendrabasnet** [comment]
Hey @samus-aran i didnt receive the coupon codes in the swag orders I placed early this month , I made threads here (https://github.com/orgs/community/discussions/48129) and (https://github.com/orgs/community/discussions/43406#discussioncomment-4972867 ) and also made a Support Ticket (#2013955) but still haven't heard anything back

**Comment by Hector1246** [comment]
Thank you Author for discussing this issue here. I had the same issue. , https://famewide.com/

**Reply by mridungupta1984** [reply]
Thank you @Hector1246 for acknowledging.

**Comment by ShahariarRabby** [comment]
Not fixed yet? Getting the same issue @github-staff

**Reply by mridungupta1984** [reply]
It will be fixed soon, there are major revamping going on (https://github.com/community/Global-Campus-Teachers/discussions/192#discussion-4878990)

**Comment by dmariaa** [comment]
Hi! Any news about this problem? I am getting the same "Unauthorized user" message, while my teacher status is approved...

Kind regards!!!

**Reply by mridungupta1984** [reply]
It will be fixed soon, there are major revamping going on (https://github.com/community/Global-Campus-Teachers/discussions/192#discussion-4878990)

**Comment by jhoonson99** [comment]
Thank you Author for discussing this issue here. I had the same issue https://hypeapk.com/

**Reply by mridungupta1984** [reply]
Kindly do not get it generated by AI.

**Comment by arias-spu** [comment]
Hello, is it fixed yet? What is the URL? I don't seem to find it. @samus-aran

**Reply by yondoi** [reply]
Have you found URL

**Comment by GitHubDAIICT** [comment]
Hello @github-staff

As a part of GitHub Campus Program, we would like to place the Github Campus Swags using the [link](https://airtable.com/shr1HLQyQM2tVuo2u). By doing so we are getting the following screen:
![image](https://github.com/community/community/assets/82431660/29b3b83a-8067-4874-aced-c06ec0632e2f)

The campus swags would be very much helpful for students in their development journey. So it would be better if you can help us out as soon as possible.

**Reply by javicatalaiesdoctorbalmis** [reply]
We have the same issue. Any feedback from GitHub?

**Comment by chakka-guna-sekhar-venkata-chennaiah** [comment]
Hello @samus-aran . As a student, how can I get the education swags which contains git cheat sheets like that???

**Comment by josegomez** [comment]
Any updates?

**Comment by abhishekkumkar** [comment]
Is this issue resolved ?

On Wed, 8 Feb 2023 at 9:31 AM, mridungupta1984 ***@***.***>
wrote:

> Hey, Community Members and @github-staff <https://github.com/github-staff>
> .
>
> *Today, I am starting this discussion to report for the issues that
> multiple users are facing (while submitting request for Swags).*
>
>    1. *Option is for requesting is no longer visible*
>
> Screenshot of the "Global Campus" for reference
> [image: issue-1_screenshot]
> <https://user-images.githubusercontent.com/119317635/217423932-22874a8d-9a2b-40c1-95cf-934c58cbfecf.png>
> [by @mridungupta1984 <https://github.com/mridungupta1984>]
>
>
>    2. *Unauthorized users at the link
>    <https://githubedu.printengine.com/product/github-teacher-toolbox-swag-offer/>
>    {general URL for requesting swags}*
>
> Screenshot of the error occurring (Courtesy to some users participated
> for other Github Discussions)
>
> [image: issue-2_screenshot1]
> <https://user-images.githubusercontent.com/119317635/217425535-118ac823-310f-4a00-8a38-09efc3f63036.png>
> [by @endersonmenezes <https://github.com/endersonmenezes>]
>
> [image: issue-2_screenshot2]
> <https://user-images.githubusercontent.com/45125059/152116951-28e70331-84a4-4788-b10f-bb0cdd1ca134.png>
> [by @abhishekkumkar <https://github.com/abhishekkumkar>]
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/46694>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKYI3Q5TYS24ZC66JYTRFELWWMLCNANCNFSM6AAAAAAUUYCFDI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
**Disclaimer:*
This message was sent from Vellore Institute of Technology.  
The contents of this email may contain legally protected confidential or 
privileged information of “Vellore Institute of Technology”.  If you are 
not the intended recipient, you should not disseminate, distribute or copy 
this e-mail. Please notify the sender immediately and destroy all copies of 
this message and any attachments. If you have received this email in error, 
please promptly notify the sender by reply email and delete the original 
email and any backup copies without reading them.*

**Reply by chakka-guna-sekhar-venkata-chennaiah** [reply]
no

**Comment by shahidhussainsoomro** [comment]
When will this issue be resolved?

**Comment by josegomez** [comment]
Is this ever going to be resolved? Or is this part of the program dead.
